java/ql/src/experimental/Security/CWE/CWE-078/CommandInjectionRuntimeExec.ql

```diff
@@ -0,0 +1,26 @@
+/**
+ * @name Command Injection into Runtime.exec() with dangerous command
+ * @description High sensitvity and precision version of java/command-line-injection, designed to find more cases of command injection in rare cases that the default query does not find
+ * @kind path-problem
+ * @problem.severity error
+ * @security-severity 6.1
+ * @precision high
+ * @id java/command-line-injection-extra
+ * @tags security
+ *       experimental
+ *       external/cwe/cwe-078
+ */
+
+import CommandInjectionRuntimeExec
+import ExecUserFlow::PathGraph
+
+class RemoteSource extends Source instanceof RemoteFlowSource { }
```

Comment:
Any reason to not include this in the `.qll` file?

Reply:
No, I'll make that change